packages/next/site/src/pages/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Content from '../components/content';
+import { Title } from '@patternfly/react-core';
+import { StyleSheet, css } from '@patternfly/react-styles';
+import packageJson from '../../../core/package.json';
+
+const styles = StyleSheet.create({
+  hero: {
+    height: '100vh',
+    backgroundColor: '#000',
+    display: 'flex',
+    alignItems: 'center'
+  },
+  heroText: {
+    color: '#fff'
+  }
+});
+
+const IndexPage = () => (
+  <div>
+    <div className={css(styles.hero)}>
+      <Content>
+        <Title size="xxxxl" withMargins className={css(styles.heroText)}>
+          Patternfly React
+        </Title>
+        <Title size="sm" className={css(styles.heroText)}>
```

Comment:
getting a prop warning in console (`sm` is not valid it seems)

Reply:
Missed this.  It was removed before publish, but existed while I was working against my local instance of pf4.